WRO-697: Create babel-preset-enact to maintain the rule in one place (#1)
enactjs/babel-preset-enact: README.md

> # babel-preset-enact

Reviewer: Can we update README with the detailed information? If it cannot be done now, let's create another ticket.

Author: OK, I will create another ticket.

Author: WRP-7819

> > A set of babel preset designed by the Enact team.

Reviewer: ```suggestion
> This package includes the Babel preset used by Enact.
```
